server/start.go

```diff
@@ -488,9 +495,66 @@ func startInProcess(svrCtx *server.Context, clientCtx client.Context, opts Start
 		return err
 	}
 
+	// Start running rpc server for sgx binary access on the evm Keeper statedb
+	listener, err := startRpcServer(svrCtx, g, app)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		// TODO: Graceful stop
+		listener.Close()
+	}()
+
 	return g.Wait()
 }
 
+func startRpcServer(
+	svrCtx *server.Context,
+	g *errgroup.Group,
+	app types.Application,
+) (listener net.Listener, err error) {
+	ethApp := app.(IncoApp)
+	if ethApp == nil {
+		return nil, nil
+	}
```

Comment:
Will these 3 lines ever happen? I think we can remove. If `app` is nil, then L517 will panic at startup, which is fine
```suggestion
```

Reply:
Agree, done.